wildfly-cloud-legacy-galleon-pack/src/main/resources/configs/standalone/standalone.xml/config.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" ?>
 
+<!-- The default config expects the wildfly-datasources-galleon-pack to be present in the provisioning config -->
```

Comment:
does that mean that wildfly-datasources-galleon-pack is a dependency of wildfly-cloud-legacy-galleon-pack?

Reply:
No dependency. Both need to provisioned if one wants to provision the default configuration identical to the one present in the image default server.